_kosmorro/main.py

```diff
@@ -53,6 +58,16 @@ def main():
 
     position = get_position(args.position) if args.position not in [None, ""] else None
 
+    # if output format is not specified, try to use output file extension as output format
+    if args.output is not None and output_format is None:
+        file_extension = os.path.splitext(args.output)[-1][1:].lower()
+        if file_extension:
+            output_format = file_extension
+
+    # default to .txt if output format was not given and output file did not have file extension
+    if output_format is None:
+        output_format = list(output_formats.keys())[0]
```

Comment:
I thought about doing this, but won't that return the text dumper instead of the string "txt"? Maybe there's another way to do this I'm not thinking of.

Reply:
Oh, you're right, it will return the dumper.
But then, why not simply use the `"txt"` string instead?

Reply:
I have made this change as well.